src/App.js

```diff
@@ -14,12 +14,21 @@ import NotFound from "./scenes/NotFound";
 import api from "./services/api";
 
 import { library } from "@fortawesome/fontawesome-svg-core";
-import { faSpinner, faTimes } from "@fortawesome/free-solid-svg-icons";
+import {
+    faSpinner,
+    faTimes,
+    faBuilding,
+    faPaperPlane,
+    faChartArea
+} from "@fortawesome/free-solid-svg-icons";
 
 import "./assets/bootstrap.css";
 
 library.add(faSpinner);
 library.add(faTimes);
+library.add(faBuilding);
```

Comment:
libraray. add supports multiple icons on the same call,

library.add(fas, far, fab)